### PR TITLE
Fix isochrones

### PR DESCRIPTION
**Pull request by l-vincent-l**
We were retrieving one response per duration.

### CODE REVIEW

File: source/routing/tests/routing_api_test.cpp [hunk 1]
```diff
@@ -1522,3 +1522,29 @@ BOOST_AUTO_TEST_CASE(use_crow_fly){
...
+
+
+    navitia::type::EntryPoint ep;
+    ep.type = navitia::type::Type_e::StopPoint;
+    ep.uri = "A1";
+
+    auto result = nr::make_isochrone(raptor, ep, navitia::test::to_posix_timestamp("20120615T082000"), true, {}, {}, sn_worker,
+                                     false, true);
+
+    std::cout << result.DebugString() << std::endl;
```
Comment (antoine-de) [inline source/routing/tests/routing_api_test.cpp]: debug cout

File: source/routing/raptor_api.cpp [hunk 2]
```diff
@@ -967,7 +967,10 @@ pbnavitia::Response make_isochrone(RAPTOR &raptor,
 
      std::sort(response.mutable_journeys()->begin(), response.mutable_journeys()->end(),
                [](const pbnavitia::Journey & journey1, const pbnavitia::Journey & journey2) {
-               return journey1.duration() < journey2.duration();
+                auto duration1 = journey1.duration(), duration2 = journey2.duration();
+                if (duration1 != duration2)
+                    return duration1 < duration2;
```
Comment (TeXitoi) [inline source/routing/raptor_api.cpp:972]: I thought bracket less if was prohibited? ;)

File: source/routing/tests/routing_api_test.cpp [hunk 3]
```diff
@@ -1522,3 +1522,27 @@ BOOST_AUTO_TEST_CASE(use_crow_fly){
...
+    b.data->build_raptor();
+    nr::RAPTOR raptor(*(b.data));
+    ng::StreetNetwork sn_worker(*b.data->geo_ref);
+
+
+    navitia::type::EntryPoint ep;
+    ep.type = navitia::type::Type_e::StopPoint;
+    ep.uri = "A1";
+
+    auto result = nr::make_isochrone(raptor, ep, navitia::test::to_posix_timestamp("20120615T082000"), true, {}, {}, sn_worker,
```
Comment (TeXitoi) [inline source/routing/tests/routing_api_test.cpp]: long line

File: source/routing/raptor_api.cpp (hunk 2 above)
Comment (l-vincent-l) [inline source/routing/raptor_api.cpp:972]: not in lambda functions.
Haven't you read the C++ guidelines?

Comment (TeXitoi) [inline source/routing/raptor_api.cpp:972]: Quote or STFU ;)

**Comment by TeXitoi** [comment]
OK for me modulo these 2 remarks.

**Comment by l-vincent-l** [comment]
I took in account your cosmetic remarks

**Pull request merged**